src/codeenginestudio/thuy/Octal.java

```diff
@@ -0,0 +1,54 @@
+package codeenginestudio.thuy;
+
+public class Octal extends Numberal {
+
+    public Octal(int number) {
+        this.setNumber(number);
+    }
+
+    @Override
+    public boolean equals(Object ob) {
+        if (this == ob) {
+            return true;
+        }
+        if (ob == null || getClass() != ob.getClass()) {
+            return false;
+        }
+        Octal octal = (Octal) ob;
+
+        return getNumber() == octal.getNumber();
+    }
+
+    @Override
+    public Octal minus(int i) {
+        return new Octal(getNumber() - i);
+    }
+
+    public Octal minus(Octal octal) {
+        return new Octal(getNumber() + octal.getNumber());
+    }
+
+    public static int parse(String ob) {
+        String replaceString = "";
+        if (ob.contains(Contants.START_LETTER)) {
```

Comment:
I think we should remove `if` condition in here. Do you think so ?

Reply:
its ok.